index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const path = require('path');
```

Comment:
Categories import statements and add a comment.
e.g. 
// Module Imports
All imports related to npm modules

// Component Imports
....

Reply:
Done!